src/test/java/HW15.java

```diff
@@ -0,0 +1,69 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class HW15 {
+
+//    @Test
+//    public static void HW15 () throws InterruptedException {
+//        WebDriver driver = new ChromeDriver();
+//        try {
+//            driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+//            String url ="https://bbb.testpro.io/";
+//            String registrationURL = "https://bbb.testpro.io/registration.php";
+//            driver.get(url);
+//            WebElement registrationField = driver.findElement(By.id("hel"));
+//            registrationField.click();
+//            Thread.sleep(2000);
+//            Assert.assertEquals(driver.getCurrentUrl(),registrationURL);
+//
+//
+//
+//        }
+//        finally {
+//            driver.quit();
+//
+//        }
+
+//    }
+    @Test
+    public static void HomeWork15 () throws InterruptedException {
+        WebDriver driver = new ChromeDriver();
```

Comment:
The text Pluto  is used 2 times!Don't repeat yourself -) I suggest to use one common variable like

`String nameSong = "Pluto";`